### PR TITLE
Add HANA System Replication (HSR)

### PR DESCRIPTION
**Pull request by xmanningcentiq**
## Add HANA System Replication (HSR) custom log

Work as part of #92 

This PR enables collection of data from the [M_SERVICE_REPLICATION](https://help.sap.com/viewer/4fe29514fd584807ac9f2a04f6754767/2.0.04/en-US/20c43fc975191014b0ece11b47a86c10.html) and [M_SYSTEM_REPLICATION](https://help.sap.com/viewer/7c78579ce9b14a669c1f3295b0d8ca16/Cloud/en-US/4b263e651f6a42cb836d7aff26894bc5.html) views to present the status of replication as a timeseries for each HANA instance and Database Tenants.

This statement is run from the SYS_DATABASES schema from SYSTEMDB, this can be queried in Log Analytics as `SapHana_SystemReplication_CL`.

### CODE REVIEW

File: sapmon/content/SapHana.json [hunk 1]
```diff
@@ -113,6 +113,22 @@
...
...MVIOS.TYPE = 'LOG' JOIN SYS_DATABASES.M_SERVICES AS MSRV ON MSRVR.PORT = MSRV.PORT AND MSRVR.HOST = MSRV.HOST INNER JOIN SYS.M_HOST_INFORMATION AS MHI ON UPPER(MHI.KEY) = 'SID' AND MHI.HOST = MSRVR.HOST GROUP BY MHI.VALUE, MSRVR.HOST, MSRVR.PORT, MSRV.SERVICE_NAME, MSRV.ACTIVE_STATUS, MSRVR.SITE_NAME, MSRVR.DATABASE_NAME, MSRVR.SECONDARY_HOST, MSRVR.SECONDARY_SITE_NAME, MSRVR.REPLICATION_MODE, MSYSR.OPERATION_MODE, MSRVR.VOLUME_ID, MSYSR.TIER, MSYSR.REPLICATION_STATUS, MSRVR.REPLICATION_STATUS, MSRVR.REPLICATION_STATUS_DETAILS, MSYSR.SECONDARY_READ_ACCESS_STATUS, MSRVR.SECONDARY_FULLY_RECOVERABLE, MSRVR.FULL_SYNC, MSRVR.SHIPPED_LOG_BUFFERS_SIZE, MSRVR.SHIPPED_LOG_BUFFERS_COUNT, MSRVR.SHIPPED_LOG_BUFFERS_DURATION, MSRVR.SHIPPED_FULL_REPLICA_SIZE, MSRVR.SHIPPED_FULL_REPLICA_DURATION, MSRVR.SECONDARY_RECONNECT_COUNT, MSRVR.SECONDARY_FAILOVER_COUNT, MSRVR.LAST_LOG_POSITION, MSRVR.LAST_LOG_POSITION_TIME, MSRVR.SHIPPED_LOG_POSITION, MSRVR.SHIPPED_LOG_POSITION_TIME, MSRVR.SHIPPED_SAVEPOINT_START_TIME, MVIOS.TOTAL_WRITE_SIZE, MVIOS.TOTAL_WRITE_TIME, MVIOS.TOTAL_TRIGGER_ASYNC_WRITES, MSRVR.LAST_LOG_POSITION, MSRVR.SHIPPED_LOG_POSITION ORDER BY MSRVR.SECONDARY_HOST DESC, MSRVR.VOLUME_ID ASC"
```
Comment (rukawata) [inline sapmon/content/SapHana.json:128]: Is this a time series query or point of time query?

Comment (xmanningcentiq) [reply sapmon/content/SapHana.json]: Hi @rukawata ,

This is point in time, I have removed the `"isTimeSeries": true,` from the query.

Comment (rukawata) [reply sapmon/content/SapHana.json]: @xmanningcentiq , thanks for clarification!
@tniek, for point in time query, do you need `isTimeSeries` set to `false` or remove it completely?

**Comment by xmanningcentiq** [comment]
Hi @tniek and @rukawata - apologies for pushing another change and subsequently dismissing your review, I have pushed a change to the SQL to gather the replayed log metrics requested by Sam.

**Pull request merged**